samples/src/Microsoft.Azure.IIoT.App/src/Pages/_DrawerDiscoverer.razor

```diff
@@ -29,20 +29,20 @@
                 <label>Max Network Probes</label>
                 <input @bind="@DiscovererData.RequestedMaxNetworkProbes" aria-label="maxNetworkProbes" id="MaxNetworkProbes" type="text" class="form-control" placeholder="@(DiscovererData.EffectiveMaxNetworkProbes ?? "Max Network Probes")" />
             </div>
-            <div class="form-group">
```

Comment:
Why the order change?

Reply:
So that the order matches the one on the assets page